packages/veritone-widgets/src/widgets/ContentTemplate/story.js

```diff
@@ -0,0 +1,185 @@
+              },
+              {
+                id: 'schemaGuid2',
+                status: 'published',
+                // dataRegistry: {
+                //   name: 'Twitter Schema 2'
+                // },
+                definition: {
+                  properties: {
+                    url: {
+                      type: 'string'
+                    },
+                    username: {
+                      type: 'string'
+                    },
+                    password: {
+                      type: 'string'
+                    }
+                  }
+                }
+              },
+              {
+                id: 'schemaGuid2',
+                status: 'published',
+                // dataRegistry: {
+                //   name: 'Twitter Schema'
+                // },
+                definition: {
+                  test: 'citest'
+                }
+              },
+              {
+                id: 'schemaGuid2',
+                status: 'draft',
+                // dataRegistry: {
+                //   name: 'Twitter Schema'
+                // },
+                definition: {
+                  properties: {
+                    url: {
+                      type: 'string'
+                    },
+                    username: {
+                      type: 'string'
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  }
+};
+
+function createTemplateData(dataSchemas) {
+  const templateSchemas = {};
+  // array of data registries containing an array of schemas
+  dataSchemas.reduce((schemaStore, registryData) => {
+    registryData.schemas.records.forEach(schema => {
+      // only take schemas that are 'published' and also define field types
+      if (
+        schema.status === 'published' &&
+        has(schema.definition, 'properties')
+      ) {
+        schemaStore[schema.id] = {
+          name: registryData.name,
+          ...schema
+        };
+      }
+    });
+  }, templateSchemas);
+
+  return templateSchemas;
+}
+
+function createInitialTemplates(templateSources) {
+  const selectedTemplateSchemas = {};
+
+  const templateSchemas = createTemplateData(
+    result.data.dataRegistries.records
+  );
+  templateSources.forEach(template => {
+    if (has(templateSchemas, template.schemaId)) {
+      selectedTemplateSchemas[template.schemaId] =
+        templateSchemas[template.schemaId];
+      if (template.data) {
+        // if we need to fill out the form with pre-data
+        selectedTemplateSchemas[template.schemaId].data = template.data;
+      }
+    }
+  });
+
+  return selectedTemplateSchemas;
+}
+
+const templateData = createTemplateData(result.data.dataRegistries.records);
+const initialTemplates = createInitialTemplates(
+  source.data.source.contentTemplates
+);
+
+class Story extends React.Component {
+  componentDidMount() {
+    this._ctFormWidget = new ContentTemplateFormWidget({
```

Comment:
handleUpdateContentTemplates is missing but is a required prop, at minimum set as noop.